dev/routing/auth.js

```diff
@@ -20,6 +21,21 @@ module.exports = function (server) {
 
   server.route({
     method: 'POST',
+    path: '/api/reset_password',
+    config: {
+      validate: {
+        payload: {
+          email: Joi.string().email().min(3).max(255).required(),
+        }
+      },
+      handler: function (request, reply) {
+        reply(resetPasswordService.resetPassword(request.payload.email));
```

Comment:
resetPasswordService.resetPassword(request.payload.email) returns promise
Are you sore that you want to reply promise?